### ISSUE TITLE
Offer release on https://jitsi.org/ ?

### ISSUE THREAD

**Issue by real-or-random** [jitsi/jitsi-meet-electron#265]
Is it possible to add a download link to  https://jitsi.org/downloads ? I think a lot of users are not aware of this app, which is very useful if they usually use Firefox/Safari or just want a better desktop integration. Independently of this, the "Jitsi Videobridge, Jitsi Meet, Jicofo and Jigasi stable build line" section should state more clearly that this is the server part of Jitsi Meet.

When I mention to people that there's a desktop app, they tend to search for it and end up on https://desktop.jitsi.org/ , which I know is the old website but maybe should have a big notice and point to Jitsi Meet more prominently.

**Comment by saghul** [comment]
We are revampting our website right this moment, so we'll take this into account, thank you!

**Comment by nathanael-h** [comment]
And maybe, also in the default page of selfhosted Jitsi Meet installation ? :wink:

**Comment by noFAYZ** [comment]
> We are revampting our website right this moment, so we'll take this into account, thank you!

The Products are awesome and but the website looks like from 90's with no clear information provided.

**Comment by jmcclelland** [comment]
We are also having trouble helping people download the jitsi meet electron app.

When we send people to the releases page in this repo, it's confusing because people don't know that "Windows" means the .exe download, Mac means the dmg etc.

On the jitsi.org download page, It's great to see the mobile jitsi-meet apps featured prominently - that is a huge win. 

In our experience, there is a lot of confusion between "Jitsi Desktop" and "Jitsi Meet". Many people are told we will be using "jitsi" to meet and they end up with the Jitsi Desktop app instead of this electron package.

I would suggest a significant division between the jitsi and jitsi-meet on the download page with the jitsi meet mobile and desktop apps featured prominently. The server components can be de-emphasized (most people who are looking for the server components know what they are looking for).

**Comment by saghul** [comment]
Yep, totally right folks. We just need some extra cycles to get to it.

**Comment by real-or-random** [comment]
I believe this is essentially solved.

**Comment by real-or-random** [comment]
Ah no, sorry! I was going through my old issues and didn't realize that this is in the electron repo. So this is not solved. :)

**Comment by samuelroland** [comment]
Any news ?? It's really important to have this electron client easily avaible for non-dev persons. My friends want to install it and I have to help them to install... Or they install jitsi instead... That doesn't take a long time to create a link on the download zone right ?? Please do it soon! Thanks for your great app.

**Comment by JobNoorman** [comment]
Sorry for the bump but is this still being planned?

I'm currently a bit embarrassed when I tell people I prefer to use Jitsi for privacy reasons but then have to say they should use Google Chrome for best quality. Being able to point them to an official client would help a lot but sending a GitHub link to non-technical users is not ideal.

**Comment by saghul** [comment]
Yes 🤦

**Comment by roaldarbol** [comment]
Hi guys! Just [posted on the community forum](https://community.jitsi.org/t/suggestion-include-electron-app-on-downloads-page/114066), then noticed this issue. I think everthing has been said here, and just to reiterate: This seems like it should be a real priority. So just bumping this again. 
Thanks for all the great work you do!
